graphqlapi/src/main/java/io/stargate/graphql/schema/schemafirst/fetchers/admin/DeploySchemaResponseDto.java

```diff
@@ -0,0 +1,64 @@
+package io.stargate.graphql.schema.schemafirst.fetchers.admin;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+import graphql.GraphqlErrorHelper;
+import graphql.schema.DataFetchingEnvironment;
+import io.stargate.graphql.schema.schemafirst.migration.MigrationQuery;
+import io.stargate.graphql.schema.schemafirst.processor.ProcessingMessage;
+import java.util.List;
+import java.util.Locale;
+import java.util.Map;
+import java.util.UUID;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class DeploySchemaResponseDto {
+
+  private UUID version;
+  private List<ProcessingMessage> messages;
+  private List<MigrationQuery> cqlChanges;
+
+  public void setVersion(UUID version) {
+    this.version = version;
+  }
+
+  public UUID getVersion() {
+    return version;
+  }
+
+  public void setMessages(List<ProcessingMessage> messages) {
+    this.messages = messages;
+  }
+
+  public List<Map<String, Object>> getMessages(DataFetchingEnvironment environment) {
+    Stream<ProcessingMessage> stream = messages.stream();
+    String category = environment.getArgument("category");
```

Comment:
maybe we should create an enum with all possible categories?

Reply:
Actually there is already one: `ProcessingMessageType`. But it also contains error statuses which are not used in this context. It would probably be cleaner to split it into two different enums, I'll give that a try.